drafts/2016-03-14-this-week-in-rust.md

```diff
@@ -53,12 +54,25 @@ If you are a Rust project owner and are looking for contributors, please submit
 
 # Updates from Rust Core
 
-113 pull requests were [merged in the last week][merged].
+142 pull requests were [merged in the last week][merged].
 
-[merged]: https://github.com/issues?q=is%3Apr+org%3Arust-lang+is%3Amerged+merged%3A2016-02-29..2016-03-07
+[merged]: https://github.com/issues?q=is%3Apr+org%3Arust-lang+is%3Amerged+merged%3A2016-03-07..2016-03-14
 
 ## Notable changes
 
+* [implement RFC 1192 inclusive ranges](https://github.com/rust-lang/rust/pull/30884)
+* [SIMD intrinsics for fused multiply-add](https://github.com/rust-lang/rust/pull/32066)
+* [AVX broadcast/conversion intrinsics](https://github.com/rust-lang/rust/pull/32140)
+* [New i586-pc-windows-msvc target](https://github.com/rust-lang/rust/pull/32034)
+* [Huge-String-Slice-Overflow-DoS averted](https://github.com/rust-lang/rust/pull/32064)
+* [Forbid having overlapping fns in inherent impls for same type](https://github.com/rust-lang/rust/pull/31925)
+* [Add Pass manager for MIR](https://github.com/rust-lang/rust/pull/31916)
+* [fn item type shenanigans](https://github.com/rust-lang/rust/pull/31710)
```

Comment:
Could be renamed - *fn item types are properly unique and have size zero*.

This PR is interesting too Add impl of FnOnce to AssertRecoverSafe https://github.com/rust-lang/rust/pull/32102